cratera/papers
# Create Text component

# Description



Create a `Text` component that uses our fonts and font sizes and replace every `react-native's` `Text` instance with it.

## How to test



Run `pnpm storybook` and check the `components/Text` story or run `pnpm start` and play around with any `Text` instance.

## Visuals


<img src="https://github.com/cratera/papers/assets/63152281/89b14fbb-da4a-43f0-bbdb-890a1b1686b9" width="360">

> Storybook `Text` screen with every variant

## Additional Notes



- We only support **bold** when using the `sans-serif` font, this is intentional afaik.
- Updated the app icon and splash image with the ones that were being used in the previous repo.
- Added our custom fonts (`Karla` and `YoungSerif`) to the `tailwind` config, our app and the storybook app.
- Added our theme colors and font sizes to the `tailwind` config.
- Added path aliases support to `jest` by mimicking the ones we have in the `tsconfig`.

---

[Notion task](https://www.notion.so/jcdvr/Task-board-caeb522237b14e1599875c72613b1c27?p=e79f6141dfc2423aa5e27a35e43a3ac3&pm=s)
